README.md

```diff
@@ -47,6 +47,10 @@ To submit iOS apps to the App Store, you must build them using Xcode 15 or later
 
 React Native 0.7x introduced support for a new architecture. We don't yet support the new architecture.
 
+## Expo Support
+
+See [expo.md](./docs/expo.md) on guidance on integrating the SDK with Expo projects.
```

Comment:
`See expo.md on ...`  
Can we rephrase this slightly and replace 'expo.md' with different words? For example: 
Please refer to the [Expo Integration](./docs/expo.md) document for guidance on integrating the SDK with Expo projects.